enhancements/analyzer-mcp/README.md

```
@@ -0,0 +1,252 @@
+
+As a DevOps engineer, I want to run analysis on pull requests.
+
+1. Deploy MCP server with HTTP transport
+2. Configure bearer token authentication
+3. CI job calls `analyze` on the PR branch
+4. Parse results and post comments on PRs with migration issues
+
+### Implementation Details/Notes/Constraints [optional]
+
+**Architecture:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)